Dockerfile.control-plane

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19 AS builder
+FROM registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.24-openshift-4.20 AS builder
```

Comment:
Does it matter here to switch from the builder to the release type? The rest of the Dockerfiles use release.

Reply:
This might be a problem. According to historical records, the release type has never been used.

Reply:
I changed it to the ocp/builder version